corehq/apps/users/templates/users/edit_commcare_user.html

```diff
@@ -70,11 +70,15 @@
           <div class="form form-horizontal">
             <fieldset>
               <legend>{% trans 'Force Full Refresh on next Mobile Sync' %}</legend>
-              <div>
+              <form action="{% url "force_user_412" domain couch_user.user_id %}" method="post">
                 <p>
-                  <a class="btn btn-primary {% if not has_any_sync_logs %} disabled{% endif %}" href="#force_user_412_{{ couch_user.user_id }}" data-toggle="modal" {% if not has_any_sync_logs %}aria-disabled="true"{% endif %}>
+                  See <a href="https://confluence.dimagi.com/display/commcarepublic/CommCare+Android+Troubleshooting#CommCareAndroidTroubleshooting-ForceFullRefresh">{% trans "Force Full Refresh" %}</a> for more information.
```

Comment:
nitpick: usually HQ opens help links in a new tab

Reply:
Wow good eye. Will update